CONTRIBUTORS

```diff
@@ -43,4 +43,5 @@ Paul Lietar <lietar@google.com>
 Pierre Phaneuf <pphaneuf@google.com>
 Rob Stradling <rob@comodo.com>
 Ruslan Kovalov <ruslan.kovalyov@gmail.com>
+Vladimir Rutsky <vladimir@rutsky.org>
```

Comment:
Is your contribution owned by one of the companies already in `AUTHORS`? If not, add yourself to `AUTHORS`, as well (see the explanation at the top of this file).